<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/file/images/14023.jpg);


</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>68</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();
</string>
      </property>
      <property name="text">
       <string>书 的 ID：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string>书    名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string>作    者：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string>网    址：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string>ISBN 10 ：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();
</string>
      </property>
      <property name="text">
       <string>ISBN 13 ：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string>页    数：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string>价    格：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();
</string>
      </property>
      <property name="text">
       <string>装帧形式：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string>出 版 社：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string>出版日期：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>59</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: \*url();

</string>
   </property>
   <property name="text">
    <string>内容简介：</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="shuID">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="title">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="author">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="alt">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="isbn10">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="isbn13">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pages">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="price">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="binding">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="publisher">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pubdate">
      <property name="styleSheet">
       <string notr="true">border-image: \*url();

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: \*url();

</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="summary">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: \*url();

</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>319</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: \*url();

</string>
   </property>
   <property name="text">
    <string>所查询书籍的信息如下：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
